xcodeproj/internal/xcodeproj_macro.bzl

```diff
@@ -57,6 +58,21 @@ def xcodeproj(
             If this is set to `"bazel"`, the project will use Bazel to build
             targets, inside of Xcode. The Xcode build system still unavoidably
             orchestrates some things at a high level.
+        config: Optional. The Bazel configuration to use when generating the
```

Comment:
Is this actually optional at this level since it defaults to `rules_xcodeproj` if unset?

Reply:
ok.

Reply:
I'm just trying to follow convention. I might be totally wrong 🤷‍♂️.

Reply:
I have no idea what the starlark convention is. I know for Swift you'd say this "defaults to `rules_xcodeproj`" rather than say it's optional.